boards/arm/stm32f334_disco/Kconfig.defconfig

```diff
@@ -0,0 +1,29 @@
+# STM32F334DISCOVERY board configuration
+
+# Copyright (c) 2017 I-SENSE group of ICCS
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_STM32F334_DISCO
+
+config BOARD
+	default "stm32f334_disco"
+
+config UART_2
+	default y
+	depends on UART_CONSOLE
+
+if I2C
```

Comment:
Use `depends on FOO` when there is only one symbol inside the `if FOO` statement.